<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WkfLogConsolePrefWidget</class>
 <widget class="QWidget" name="WkfLogConsolePrefWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>控制台</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QSpinBox" name="fontSizeSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="fontSizeLabel">
       <property name="text">
        <string>字体大小:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lineCapLabel">
       <property name="text">
        <string>线帽:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="lineCapSpinBox">
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="singleStep">
        <number>100</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextBrowser" name="exampleText">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>64</height>
      </size>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Example Font Size&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="explanation2">
     <property name="text">
      <string>优先级按照颜色列表的顺序依次递增.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="colorPatternsGroupBox">
     <property name="title">
      <string>颜色模式</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>440</width>
           <height>194</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="patternList">
          <item>
           <widget class="QPushButton" name="appendColorButton">
            <property name="text">
             <string>增加颜色</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="regexHelpLabel">
       <property name="text">
        <string>正则表达式帮助:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="regexHelpButton">
       <property name="toolTip">
        <string>正则表达式帮助</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="question">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fontSizeSpinBox</tabstop>
  <tabstop>lineCapSpinBox</tabstop>
  <tabstop>exampleText</tabstop>
  <tabstop>colorPatternsGroupBox</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>appendColorButton</tabstop>
  <tabstop>regexHelpButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
